### PR TITLE
Feature/styling

### PR DESCRIPTION
**Pull request by lmcallister20**
For nav bar features, tests written

### CODE REVIEW

File: pickup/urls.py [hunk 1]
```diff
@@ -4,6 +4,8 @@
 
 urlpatterns = [
     path('', views.index, name='index'),
+    path('home/', views.home, name='home'),
+    path('profile/', views.profile_list, name='Profile List'),
```
Comment (jgalita1) [inline pickup/urls.py:8]: This line (8) should be removed. That view is no longer used. There is another profile view that is below.

Comment (lmcallister20) [reply pickup/urls.py]: Thank you so much!

File: pickup/navbar_tests.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+from django.test import TestCase
+
+class navbar_tests(TestCase):
+	def bar_exists_test(self):
+		        response = self.client.get('/home/')
+        		self.assertEqual(response.status_code, 200)
```
Comment (jgalita1) [inline pickup/navbar_tests.py:6]: It would be good to have four spaces of indentation here.

File: pickup/templates/pickup/base.html [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+<html lang="en">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <!-- Bootstrap CSS -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+
+    <title>Hello, world!</title>
```
Comment (jgalita1) [inline pickup/templates/pickup/base.html]: Ideally, this should be customizable for each page, but saying "Pickup Games" would be sufficient for now.

File: pickup/templates/pickup/base.html [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js" integrity="sha384-ka7Sk0Gln4gmtz2MlQnikT1wXgYsOg+OMhuP+IlRH9sENBO0LRn5q+8nbTov4+1p" crossorigin="anonymous"></script>
+
+    <!-- Option 2: Separate Popper and Bootstrap JS -->
+    <!--
+    <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.10.2/dist/umd/popper.min.js" integrity="sha384-7+zCNj/IqJ95wo16oMtfsKbZ9ccEh31eOz1HGyDuCQ6wgnyJNSYdrPa03rtR1zdB" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.min.js" integrity="sha384-QJHtvGhmr9XOIpI6YVutG+2QOK9T+ZnN4kzFN1RtK3zEFEIsxhlmWl5/YESvpZ13" crossorigin="anonymous"></script>
+    -->
+  </body>
+  <footer>
+  <p>Sample Footer<br>
```
Comment (jgalita1) [inline pickup/templates/pickup/base.html:31]: Ideally replace <br> with </p>.

File: pickup/templates/pickup/base.html [hunk 5]
```diff
@@ -0,0 +1,34 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <!-- Bootstrap CSS -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+
```
Comment (jgalita1) [inline pickup/templates/pickup/base.html:10]: I would add a link to Joseph's stylesheet here.

File: pickup/views.py [hunk 6]
```diff
@@ -7,13 +7,21 @@
 from django.contrib.auth import login
 from django.http import HttpResponse, HttpResponseRedirect
 
+from django.http import HttpResponse
+from .models import Profile
+from django.shortcuts import render
```
Comment (jgalita1) [inline pickup/views.py:12]: These lines are all redundant/

File: pickup/views.py [hunk 7]
```diff
@@ -7,13 +7,19 @@
...
+
 # Import models and forms
 from .forms import ParkForm, RegistrationForm, ProfileForm, ScheduleForm
 from .models import Profile, Player, Parks, Schedule
 
 def index(request):
     return render(request, "pickup/index.html")
-
+  
+def home(request):
```
Comment (jgalita1) [inline pickup/views.py:21]: We want login to be required to access the home page.

**Review by jgalita1 — APPROVED** [review]
Discussed via Discord 11/4. Made some changes to normalize indentation, etc. Discussed some changes (and left comments) on things that should be implemented as a part of the styling story in the next iteration.

Looks good, approved.

**Comment by lmcallister20** [comment]
All done! Checked by Josh Nov 4th

**Pull request merged**